### ISSUE TITLE
Don't show update notification every time I reload a window, but only once per boot

### ISSUE THREAD

**Issue by LinqLover** [microsoft/vscode#111185]
In my usual workflow, I open and close dozens of VS Code every day. Every time I do so, VS Code shows it "Please restart for updates" notification again. But because there are still some other windows open, I won't do so before evening. Is there an option to make the update notifications a bit more quite? They should show be displayed only if the first VS Code process is started. If there is not yet such an option, this would be a great addition.

**Comment by joaomoreno** [comment]
Actually this notification only comes up 5 days after you were first informed that there was an update. Can you confirm that?

Is this really happening often to you, that you keep Code running for longer than 5 days without updating, even after getting an update notification?

**Comment by LinqLover** [comment]
> Actually this notification only comes up 5 days after you were first informed that there was an update. Can you confirm that?

Sounds plausible.

> Is this really happening often to you, that you keep Code running for longer than 5 days without updating, even after getting an update notification?

It is ... Restarting VS Code can take a while for me, mainly because windows are not restored on the correct virtual desktop (at least without [Videfix](https://github.com/LinqLover/Videfix)) and I do have to rearrange every single code window manually after each restart. I try to minimize the number of times where that's necessary.

**Comment by joaomoreno** [comment]
One suggestion is to change the way Code updates for you: by changing `update.mode` to `manual`, Code will only check for updates if you run `Check for Updates`. Does that help you?

**Comment by LinqLover** [comment]
Hm, still not perfect. If my machine is rebooted anyway (e.g. thanks to Windows updates), I would like VS Code to have been updated, too.

**Comment by joaomoreno** [comment]
> Hm, still not perfect. If my machine is rebooted anyway (e.g. thanks to Windows updates), I would like VS Code to have been updated, too.

That's a separate feature of VS Code which isn't properly implemented even with `update.mode` set to `default`.

**Comment by LinqLover** [comment]
Please reopen, this issue has not yet been resolved. What information is still needed here?
